File: sdks/java/examples/SendSms.java
import com.pushfy.ApiException;
import com.pushfy.Pushfy;
import com.pushfy.RateLimitException;

/**
 * Send an SMS with the Pushfy Java SDK.
 *
 * <p>Compile against the built jar (or the compiled classes) and run with
 * {@code PUSHFY_API_TOKEN} in your environment:
 *
 * <pre>
 *   javac -cp target/pushfy-1.0.0.jar examples/SendSms.java -d out-examples
 *   java  -cp target/pushfy-1.0.0.jar:out-examples SendSms
 * </pre>
 */
public class SendSms {

    public static void main(String[] args) {
        String token = System.getenv("PUSHFY_API_TOKEN");
        Pushfy pushfy = Pushfy.builder().apiToken(token).build();

        try {
            Object result = pushfy.sendSms(
                    "5511999999999",
                    "Hello from the Pushfy Java SDK",
                    "demo-" + System.currentTimeMillis());
            System.out.println("Accepted: " + result);
        } catch (RateLimitException err) {
            System.err.println("Rate limited — back off and retry.");
            System.exit(1);
        } catch (ApiException err) {
            // 5xx / network — safe to retry idempotently (reuse the same extId).
            System.err.println("Transient failure: " + err.status + " " + err.getMessage());
            System.exit(1);
        } catch (Exception err) {
            System.err.println("Failed: " + err.getMessage());
            System.exit(1);
        }
    }
}

File: sdks/dotnet/examples/SendSms.cs
using System;
using System.Threading.Tasks;
using Pushfy;

// Send an SMS with the Pushfy .NET SDK.
//   dotnet run --project examples
// Requires PUSHFY_API_TOKEN in your environment.
internal static class SendSmsExample
{
    public static async Task Main()
    {
        var pushfy = new PushfyClient(new PushfyClientOptions
        {
            ApiToken = Environment.GetEnvironmentVariable("PUSHFY_API_TOKEN"),
        });

        try
        {
            var result = await pushfy.SendSmsAsync(
                to: "5511999999999",
                text: "Hello from the Pushfy .NET SDK",
                extId: "demo-" + DateTimeOffset.UtcNow.ToUnixTimeSeconds());

            Console.WriteLine("Accepted: " + result);
        }
        catch (RateLimitException)
        {
            Console.Error.WriteLine("Rate limited — back off and retry.");
            Environment.Exit(1);
        }
        catch (PushfyException err)
        {
            Console.Error.WriteLine($"Failed: {err.Status} {err.Code} {err.Response}");
            Environment.Exit(1);
        }
    }
}
